chipcie/templates/hpa.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
```

Comment:
I would recommend to remove this file since we don't use this feature to maintain simplicity.

Reply:
fixed